mythril/laser/ethereum/instructions.py

```diff
@@ -1004,7 +1004,7 @@ def sstore_(self, global_state: GlobalState) -> List[GlobalState]:
             keccak_keys = filter(keccak_function_manager.is_keccak, storage_keys)
 
             results = []
-            new = False
+            new = Bool(False)
```

Comment:
We should use the symbol factory for this. But it doesn't have a method for Bool yet, could you add that?

Reply:
Done